package.json

```diff
@@ -58,13 +58,14 @@
     "test": "gulp"
   },
   "dependencies": {
-    "@types/node": "^14.14.14",
+    "axios": "^0.19.2",
     "base-64": "^0.1.0",
     "build-url": "^1.0.10",
+    "got": "^11.8.1",
```

Comment:
we are not using GOT rite, shall we remove this?

Reply:
we have already removed this GOT package